docs/source/using-discovery/usingslurm.rst

```diff
@@ -80,21 +79,84 @@ Account information
 
 Some Discovery users have more than one Discovery group account associated with their username. For example, a student might be in a class that is using Discovery,
 and also be in a student club that is using Discovery for a club project. In this case, the student would have two group accounts associated with their username.
-When running a job with either ``srun`` or ``sbatch``, if you have more than one account associated with your username, we recommend that you use the ``--account=`` flag and specify the account
-that corresponds to the project you are working on. In the example with a student associated with a class and a student club, if the student is on Discovery submitting a job for a project
-for his or her class, set the ``account=`` flag to the name of the class account. If the student is working on a project for the club, set the ``account=`` flag to the name of the student club account.
 
-To find out what account(s) your usesrname is associated with, use the following command::
+When running a job with either ``srun`` or ``sbatch``, if you have more than one account associated with your username, we recommend that you use the ``--account=`` flag and specify the account that corresponds to the project you are working on. In the example with a student associated with a class and a student club, if the student is on Discovery submitting a job for a project for his or her class, set the ``account=`` flag to the name of the class account. If the student is working on a project for the club, set the ``account=`` flag to the name of the student club account.
 
-  sacctmgr show associations user=<yourusername>
+To find out what account(s) your username is associated with, use the following command::
 
+  sacctmgr show associations user=<yourusername>
 After you have determined what accounts your username is associated with, if you have more than one account association, you can use the ``account=`` flag with your usual ``srun`` or ``sbatch`` commands.
 
 For example, if you are associated with an account named ``dataclub`` and an account named ``info7500``, and you're currently doing work that should be associated with the
 ``dataclub`` account, in your ``srun`` command, you can add the ``--account=dataclub`` flag to specify that account.::
 
   srun --account=dataclub --partition=short --nodes=1 --ntasks=28 --mem=0 --pty /bin/bash
-
 .. note::
    If you do not have more than one account associated with your username, you do not need to use the ``--account=`` flag. Most users on Discovery have only one account
    associated with their username.
+
+SLURM Commands
+==============
+**Controlling Jobs**
+
+``scontrol hold <jobid>`` Place a hold on a pending job
+
+``scontrol release <jobid>`` Release a held job
+
```

Comment:
I never actually heard of these functions of scontrol. Are these commands that could work well within in a workflow? I am honestly trying to envision some because examples might want to be given if you state the commands.

Reply:
Yes. Updated info based on man pages